cpu/cc26x0/periph/rfc.c

```diff
@@ -104,6 +103,8 @@ void rfc_beacon(void)
 
 void rfc_prepare(void)
 {
+    printf("===> %s <===\n",__FUNCTION__);
```

Comment:
I've added a dedicated commit that include this line as `DEBUG()`. It could potentially include other output messages that are not necessary for every rfcore boot.